notebooks/406-3D-pose-estimation-webcam/README.md

```diff
@@ -0,0 +1,20 @@
+# 3D Human Pose Estimation with OpenVINO 
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/spencergotowork/openvino_notebooks.git/master?labpath=notebooks%2F224-human-pose-estimation-3d%2F)
```

Comment:
This binder link is incorrect. Please remove it. We will add the proper one later.